build.gradle

```diff
@@ -33,10 +33,9 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-jdbc'
     implementation 'org.springframework.boot:spring-boot-starter-jooq'
     implementation 'org.springframework.boot:spring-boot-starter-quartz'
-    implementation 'org.springframework.boot:spring-boot-starter-security'
-    implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
     implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation 'org.thymeleaf.extras:thymeleaf-extras-springsecurity5'
+    implementation 'org.springframework.cloud:spring-cloud-starter-openfeign:3.1.0'
```

Comment:
don't specify version, use spring dependency management

Reply:
LostfilmDemo:test: Could not find org.springframework.cloud:spring-cloud-starter-openfeign:.
Required by:
    project :

Reply:
you need to add spring cloud dependency management config, you can generate new project using spring.initializer with feign and copy config from there